src/egon/data/datasets/power_plants/__init__.py

```diff
@@ -1099,7 +1107,96 @@ def fill_missing_bus_and_geom(gens, carrier):
     logging.info(
         f"""
           {len(wind_onshore)} wind_onshore generators with a total installed capacity of
-          {wind_onshore.capacity.sum()}MW were inserted in db
+          {wind_onshore.capacity.sum()}MW were inserted into the db
+          """
+    )
+
+    # Write wind_offshore power plants in supply.egon_power_plants
```

Comment:
In general, it is a bit misleading that there is a file/module called "wind_offshore" that is now only used for future scenarios. 
It also looks like there is very simal code related to wind offshore now in two different functions: in power_plants.insert_status_quo and in power_plants.wind_offshore.insert. 
It would be better to unify that and have one function that inserts the data for wind offshore for all scenarios.